lib/utils/__tests__/addSemicolonForEditInfo.test.mjs

```diff
@@ -181,9 +192,10 @@ describe('addSemicolonForEditInfo', () => {
 
 			expect(fixData.text).toBe('@layer foo');
 
-			const text = addSemicolonForEditInfo(node, fixData).text;
-
-			expect(text).toBe('@layer foo');
+			expect(addSemicolonForEditInfo(node, fixData)).toEqual({
+				range: [0, 0],
+				text: '@layer foo;',
```

Comment:
It was previously `@layer foo` and it's now `@layer foo;`.
Should you use `toMatchObject` instead of `toEqual`?